v2/lints/etsi/lint_policy_and_qctype_are_consistent_test.go

```diff
@@ -0,0 +1,165 @@
+		},
+		{
+			Name:           "QcStmtPsd2Cert81EuQcCertWoAnyPolicy.pem",
+			InputFilename:  "QcStmtPsd2Cert81EuQcCertWoAnyPolicy.pem",
+			ExpectedResult: lint.Error,
+		},
+		{
+			Name:           "QcStmtPsd2Cert82EuQcCertWoAnyPolicy.pem",
+			InputFilename:  "QcStmtPsd2Cert82EuQcCertWoAnyPolicy.pem",
+			ExpectedResult: lint.Error,
+		},
+		{
+			Name:           "QcStmtPsd2Cert83EuQcCertWoAnyPolicy.pem",
+			InputFilename:  "QcStmtPsd2Cert83EuQcCertWoAnyPolicy.pem",
+			ExpectedResult: lint.Error,
+		},
+		{
+			Name:           "QcStmtPsd2Cert84InconsistentPolicy.pem",
+			InputFilename:  "QcStmtPsd2Cert84InconsistentPolicy.pem",
+			ExpectedResult: lint.Error,
+		},
+		{
+			Name:           "QcStmtPsd2Cert85InconsistentPolicy.pem",
+			InputFilename:  "QcStmtPsd2Cert85InconsistentPolicy.pem",
+			ExpectedResult: lint.Error,
+		},
+		{
+			Name:           "QcStmtPsd2Cert86ValidWebPolicyConsistent.pem",
+			InputFilename:  "QcStmtPsd2Cert86ValidWebPolicyConsistent.pem",
+			ExpectedResult: lint.Pass,
+		},
+		{
+			Name:           "QcStmtPsd2Cert87EuQcCertWoQcPolicy.pem",
+			InputFilename:  "QcStmtPsd2Cert87EuQcCertWoQcPolicy.pem",
+			ExpectedResult: lint.Error,
+		},
+		{
+			Name:           "QcStmtPsd2Cert88EuQcCertWoQcPolicy.pem",
+			InputFilename:  "QcStmtPsd2Cert88EuQcCertWoQcPolicy.pem",
+			ExpectedResult: lint.Error,
+		},
+		{
+			Name:           "QcStmtPsd2Cert89ValidQcTypeEsign.pem",
+			InputFilename:  "QcStmtPsd2Cert89ValidQcTypeEsign.pem",
+			ExpectedResult: lint.Pass,
+		},
+		{
+			Name:           "QcStmtPsd2Cert90MissingQcSscdStmt.pem",
+			InputFilename:  "QcStmtPsd2Cert90MissingQcSscdStmt.pem",
+			ExpectedResult: lint.Error,
+		},
+		{
+			Name:           "QcStmtPsd2Cert91InconsistentPolicy.pem",
+			InputFilename:  "QcStmtPsd2Cert91InconsistentPolicy.pem",
+			ExpectedResult: lint.Error,
+		},
+		{
+			Name:           "QcStmtPsd2Cert92InconsistentPolicy.pem",
+			InputFilename:  "QcStmtPsd2Cert92InconsistentPolicy.pem",
+			ExpectedResult: lint.Error,
+		},
+		{
+			Name:           "QcStmtPsd2Cert93ValidEsealPolicyConsistent.pem",
+			InputFilename:  "QcStmtPsd2Cert93ValidEsealPolicyConsistent.pem",
+			ExpectedResult: lint.Pass,
+		},
+		{
+			Name:           "QcStmtPsd2Cert94MissingQcSscdStmt.pem",
+			InputFilename:  "QcStmtPsd2Cert94MissingQcSscdStmt.pem",
+			ExpectedResult: lint.Error,
+		},
+		{
+			Name:           "QcStmtPsd2Cert95InconsistentPolicy.pem",
+			InputFilename:  "QcStmtPsd2Cert95InconsistentPolicy.pem",
+			ExpectedResult: lint.Error,
+		},
+		{
+			Name:           "QcStmtPsd2Cert96InconsistentPolicy.pem",
+			InputFilename:  "QcStmtPsd2Cert96InconsistentPolicy.pem",
+			ExpectedResult: lint.Error,
+		},
+		{
+			Name:           "QcStmtPsd2Cert97InconsistentPolicy.pem",
+			InputFilename:  "QcStmtPsd2Cert97InconsistentPolicy.pem",
+			ExpectedResult: lint.Error,
+		},
+		{
+			Name:           "QcStmtPsd2Cert98InconsistentPolicy.pem",
+			InputFilename:  "QcStmtPsd2Cert98InconsistentPolicy.pem",
```

Comment:
It would be useful to name 96-98 to highlight their uniqueness

Reply:
Done.